app/controllers/api_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ApiController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  private
+  def authenticated?
```

Comment:
Nice! If you have extra time, look into `OAuth` and JWT's. This is the standard for authenticating right now.